magento/pwa-studio
# commit hash at bundle banner

## Description

Adding current version in bundle generated
## Related Issue





Closes #1227 

## Acceptance 



### Verification Stakeholders

@supernova-at 

### Specification



### Verification Steps


1. Run `yarn build`
2. Open `packages/venia-concept/dist/client.[hash].js`
3. Verify that it contains a comment at the top that starts with `@version` (ex: `@version v4.0.0-106-g2eceeb7b2`).

## Screenshots / Screen Captures (if appropriate)
![Screenshot 2019-10-11 at 11 01 50 AM](https://user-images.githubusercontent.com/13298685/66626833-51a9db80-ec17-11e9-8228-61b231eea285.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
